### PR TITLE
Re-submitting changes for 1.9 missing due to forgotten last PR

### PR DESCRIPTION
**Pull request by iionly**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,24 +1,24 @@
 ### Login As 1.9.0 - February 18, 2015 ###
- * Updated for Elgg 1.9 (basically just tested it still works and it does)
- * Added German translations
- * Added Finnish translations
+ * Updated for Elgg 1.9 (basically just tested it still works and it does),
+ * Added German translations,
+ * Added Finnish translations.
```
Comment (juho-jaakkola) [inline CHANGES.txt]: This is using list syntax so there's no need for trailing commas or dots. This file should be converted to markdown syntax anyway, and the commas would look weird at the end of each row.

File: manifest.xml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <plugin_manifest xmlns="http://www.elgg.org/plugin_manifest/1.8">
 	<name>Login As</name>
+	<id>login_as</id>
 	<author>Brett Profitt</author>
 	<version>1.9.0</version>
 	<category>admin</category>
+	<category>bundled</category>
```
Comment (juho-jaakkola) [inline manifest.xml]: The plugin won't be bundled until Elgg 1.11, so I don't this this should be here for the 1.9 version.

**Comment by iionly** [comment]
Well, with login_as now being added to core and now having its own repo at Elgg/login_as I guess this PR has been forgotten once more. I won't make the same PR a 3rd time...

**Comment by juho-jaakkola** [comment]
Sorry, I didn't know you had updated the PR. Repository owner does not get a notification when you do a rebase to an existing PR. You should always add a comment manually to let people know about.

**Pull request closed without merging**